### PR TITLE
Fix mvn test for jsonpath.

### PR DESCRIPTION
**Pull request by ravitejakula**

### CODE REVIEW

File: json-path/src/main/java/com/jayway/jsonpath/internal/filter/FilterCompiler.java [hunk 1]
```diff
@@ -210,23 +209,6 @@ private RelationalExpressionNode readExpression() {
         return new RelationalExpressionNode(left, operator, right);
     }
 
-    private LogicalOperator readLogicalOperator(){
-        int begin = filter.skipBlanks().position();
-        int end = begin+1;
-
```
Comment (krdev) [inline json-path/src/main/java/com/jayway/jsonpath/internal/filter/FilterCompiler.java]: Please put the code back, why are you changing code? just add coverage...
Same comment for all source code changes, I don't expect any changes.

File: json-path-assert/pom.xml [hunk 2]
```diff
@@ -94,11 +94,6 @@
                 <version>1.3</version>
             </dependency>
             <dependency>
-                <groupId>junit</groupId>
-                <artifactId>junit</artifactId>
```
Comment (krdev) [inline json-path-assert/pom.xml:98]: why are you removing junit?

File: json-path/src/test/java/com/jayway/jsonpath/MultiPropTest.java [hunk 3]
```diff
@@ -180,41 +181,78 @@ public void deep_scan_throws_exception_readRoot() {
...
-    public void deep_scan_returns_empty_map_readRoot_with_supress_exceptions() {
+    public void deep_scan_does_not_throw_exception_readRoot_with_supressExceptions() {
     	final String json = "{\"v\": [[{}, 1, {\"a\": {\"v\": 5}, \"b\": {\"v\": 4}, \"c\": {\"v\": 1, \"flag\": true}}]]}";
-    	 Configuration conf = Configuration.builder().options(Option.SUPPRESS_EXCEPTIONS).build();
-    	 DocumentContext dc = using(conf).parse(json);
-    	 Object root = dc.readRoot(new String[] { "$..['a', 'c'].v" });
-    	 Assert.assertTrue(conf.jsonProvider().isMap(root));
-    	 Assert.assertEquals(root.toString(), "{}");
+    	Configuration conf = Configuration.builder().options(Option.SUPPRESS_EXCEPTIONS).build();
+    	Object root = using(conf).parse(json).readRoot(new String[] { "$..['a', 'c'].v" });
```
Comment (krdev) [inline json-path/src/test/java/com/jayway/jsonpath/MultiPropTest.java:187]: Add an assert statement.

Comment (ravitejakula) [reply json-path/src/test/java/com/jayway/jsonpath/MultiPropTest.java:187]: Done

File: json-path-assert/pom.xml (hunk 2 above)
Comment (ravitejakula) [reply json-path-assert/pom.xml:98]: It is in the dependencies list twice.

**Pull request merged**